Rakefile

```diff
@@ -45,6 +45,16 @@ task :default do
   sh %{rake -T}
 end
 
+namespace :pl_ci do
+  desc 'Build puppet gems'
+  task :gem_build do
+    sh "gem build puppet.gemspec --platform x86-mingw32"
```

Comment:
What happens when the platform is not `x86-mingw32`, like if we were to run this rake task on `redhat8`? Would these just fail? That might be a reasonable thing to allow so we can just ship `*.gem`, but it might obfuscate errors. I used `Open3` to capture the stdout, stdin, and status in facter's gem building, but I'm not sure if we want the CI jobs to be smart about what gets built, or fail out from here if the gem building fails.

Reply:
Oh, I guess your earlier comment answered my question about building gems for different platforms; I guess that just leaves the question about how to fail out if the gem build errors; I think you might want to copy the strategy from the [facter PR](https://github.com/puppetlabs/facter/pull/2706)?